cost-analyzer/templates/_helpers.tpl

```diff
@@ -945,9 +915,9 @@ Begin Kubecost 2.0 templates
   volumeMounts:
     - name: persistent-configs
       mountPath: /var/configs
-    {{- if or (.Values.kubecostModel).federatedStorageConfigSecret (.Values.kubecostModel).federatedStorageConfig }}
+    {{- if or (.Values.global).federatedStorage.existingSecret (.Values.global).federatedStorageConfig }}
     - name: federated-storage-config
-      mountPath: /var/configs/etl
+      mountPath: /var/configs/etl/federated
```

Comment:
According to the diff, it looks like you updated the `mountPath`? Is there a reason why?

Reply:
I bet this was an inadvertent tab completion :(